### PR TITLE
Added recipe for botocore

### PR DESCRIPTION
**Pull request by pmlandwehr**
Getting a conda-forge build of `awscli` requires a more current build of `botocore` than the main anaconda build.

### CODE REVIEW

File: recipes/botocore/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - jmespath >=0.7.1,<1.0.0
```
Comment (jakirkham) [inline recipes/botocore/meta.yaml]: Should we add a recipe for `jmespath`?

File: recipes/botocore/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - python-dateutil >=2.1,<3.0.0
+    - docutils >=0.10
```
Comment (jakirkham) [inline recipes/botocore/meta.yaml]: Should we add a recipe for `docutils`?

File: recipes/botocore/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  run:
+    - python
+    - jmespath >=0.7.1,<1.0.0
+    - python-dateutil >=2.1,<3.0.0
+    - docutils >=0.10
+    - ordereddict ==1.1  # [py26]
+    - simplejson ==3.3.0  # [py26]
+
+test:
+  # Python imports
```
Comment (jakirkham) [inline recipes/botocore/meta.yaml]: Can drop this comment.

File: recipes/botocore/meta.yaml (hunk 2 above)
Comment (pmlandwehr) [inline recipes/botocore/meta.yaml]: See #975

File: recipes/botocore/meta.yaml (hunk 1 above)
Comment (pmlandwehr) [inline recipes/botocore/meta.yaml]: See #976

**Comment by pmlandwehr** [comment]
(Ping @ocefpaf)

**Comment by jakirkham** [comment]
@kyleknap @jamesls @JordonPhillips, just letting you know we have added `botocore` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge botocore`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/botocore-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `botocore`. 😄

**Pull request merged**